lib/client/util/__mocks__/Geolocation.js

```diff
@@ -0,0 +1,37 @@
+var path = require.requireActual('../Geolocation');
```

Comment:
`path` doesn't look to be used, does this expression have done kind of side effect?